From bf6e9d06486b4c8f30d87a2b463992c4b892711e Mon Sep 17 00:00:00 2001
From: Petr Vorel <petr.vorel@gmail.com>
Date: Fri, 22 May 2020 11:14:51 +0200
Subject: [PATCH] helper: Fix missing space separator for ssh_args

Docs suggest using ssh_args without space at the end
ssh_args = "-x -o ConnectTimeout=10"

+ remove also trailing space in another line.

Fixes: 82f8845 ("Add in 'command_pre' and 'command_post' configs")

Signed-off-by: Petr Vorel <petr.vorel@gmail.com>
[ Upstream-status: https://github.com/duncs/clusterssh/pull/133 ]
---
 lib/App/ClusterSSH/Helper.pm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/App/ClusterSSH/Helper.pm b/lib/App/ClusterSSH/Helper.pm
index dbc5fb6..463a538 100644
--- a/lib/App/ClusterSSH/Helper.pm
+++ b/lib/App/ClusterSSH/Helper.pm
@@ -71,9 +71,9 @@ sub script {
            my \$user=shift;
            my \$port=shift;
            my \$mstr=shift;
-           my \$command="$command_pre $comms $comms_args";
+           my \$command="$command_pre $comms $comms_args ";
            open(PIPE, ">", \$pipe) or die("Failed to open pipe: \$!\\n");
-           print PIPE "\$\$:\$ENV{WINDOWID}" 
+           print PIPE "\$\$:\$ENV{WINDOWID}"
                or die("Failed to write to pipe: $!\\n");
            close(PIPE) or die("Failed to close pipe: $!\\n");
            if(\$svr =~ m/==\$/)
-- 
2.26.2

